wrapper/src/main/java/software/amazon/jdbc/plugin/DefaultConnectionPlugin.java

```diff
@@ -78,7 +84,6 @@ public DefaultConnectionPlugin(
       final ConnectionProvider defaultConnProvider,
       final PluginManagerService pluginManagerService,
```

Comment:
This line is missing. It might be a rebase issue.

final @Nullable ConnectionProvider effectiveConnProvider,